src/exercises/buffer-overflow-global/README.md

```diff
@@ -0,0 +1,50 @@
+# Exercise an inter-global-object buffer overflow
+
+This exercise demonstrates an inter-object buffer overflow on baseline and
```

Comment:
This paragraph should probably mention globals, even though it's in the title and below as well.